hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -358,6 +380,48 @@ private void announce(HollowConsumer.ReadState readState) {
         }
     }
 
+    private class WriteEngineWrapper {
```

Comment:
I agree a better name for this class would be good.  

The purpose was to encapsulate classes that needs to be changed on a Restore.  Originally I was going to use WriteState but that is already used.   

Let's try to come up with something better.